mlflow/utils/autologging_utils/__init__.py

```diff
@@ -346,6 +347,13 @@ def autolog(*args, **kwargs):
             config_to_store.update(kwargs)
             AUTOLOGGING_INTEGRATIONS[name] = config_to_store
 
+            # If disabling autologging using fluent api, then every active integration's autolog
+            # needs to be called with disable=True. So do not short circuit and let `mlflow.autolog()` envoke
+            # all active integrations with disable=True.
+            if name != "mlflow" and get_autologging_config(name, "disable", True):
```

Comment:
@dbczumar this is currently being tested in test_fluent_autolog.py 

[example: ](https://github.com/mlflow/mlflow/blob/bec8c2c60786ea997f6d36d2d7c5867ef1d0f8ad/tests/tracking/fluent/test_fluent_autolog.py#L162)
```
mlflow.autolog(disable=True)
 assert autologging_is_disabled(mlflow.spark.FLAVOR_NAME)
```

Reply:
Fantastic!